src/plugins/plot/stackedPlot/StackedPlotItem.vue

```diff
@@ -161,12 +161,17 @@ export default {
                         loading: true
                     };
                 },
+                computed: {
+                    isMissing() {
+                        return openmct.objects.isMissing(object);
```

Comment:
I think you can just pass this in as a data variable? I don't think it's state would be changed mid-use and if it was I don't think the code is set up in a way that it would trigger this computed variable update. When you assign object above, can you also just se the the isMissing value as well?